PSKoans/Public/Measure-Karma.ps1

```diff
@@ -90,6 +98,19 @@ function Measure-Karma {
                 Write-Verbose "Karma: $KoansPassed"
                 if ($PesterTests.FailedCount -gt 0) {
                     Write-Verbose "Your karma has been damaged."
+                    if ($ShowHelp) {
+                        $KoanAttribute = (Get-Command $KoanFile).ScriptBlock.Attributes | Where-Object TypeID -match 'KoanAttribute'
+                        Write-Host ($KoanAttribute | Out-String)
+                        if ($KoanAttribute.HelpURL) {
+                            Write-Verbose "Confusion is only the beginning."
+                            start $KoanAttribute.HelpURL
+                        }
+                        if ($KoanAttribute.HelpAction) {
+
+                            Invoke-Command $KoanAttribute.HelpAction
```

Comment:
That's an interesting thought indeed, but I'd probably prefer to simply `&` this or `.InvokeReturnAsIs()` since it's a script block being executed locally.

Reply:
Also, it looks like you removed the HelpAction in the koan class. Either remove this or add it into the koan class. 😄